install.sh

```diff
@@ -37,10 +37,13 @@ if contains $CWD "testcaser"
         ln -s "$CWD/testcaser" /usr/include/
         echo "3. Symlink created to /usr/include/"
 
-    else
+    elif [[ $platform == 'MacOS' ]]; then
```

Comment:
Convert this to else only. If platform is not Darwin or Linux we already show OS not support on line 26, that will exit the script and control never reaches that else part at 44 line. So remove that, make it as it was already. Code redundancy isn't a good practice.